modules/navier_stokes/doc/content/source/fvkernels/PINSFVMomentumFriction.md

```diff
@@ -3,32 +3,54 @@
 This kernel adds the friction term to the porous media Navier Stokes momentum
 equations. This kernel must be used with the canonical PINSFV variable set,
 e.g. pressure and superficial velocity, and supports Darcy and
-Forchheimer friction models:
+Forchheimer friction models in two flavors:
+
+## Standard friction formulation
+
+Set parameter: "standard_friction_formulation = true"
```

Comment:
Us the `[!param](parameter path)` syntax so that when people change the parameter name in the source code the documentation alerts everyone that it needs to be changed too. You can just search for the syntax above for examples.

Reply:
done